File: gradle_boot_server/src/main/java/net/canway/service/UserService.java
package net.canway.service;

import net.canway.pojo.tables.pojos.User;

import java.util.ArrayList;
import java.util.List;

public interface UserService {
    List findAllUser();

    int deleteByPrimaryKey(ArrayList<Integer> tIds);


    int insert(User record);

    int insertSelective(User record);


    User selectByPrimaryKey(Integer tId);


    int updateByPrimaryKeySelective(User record);


    int updateByPrimaryKey(User record);
}

File: gradle_boot_server/src/main/java/net/canway/service/UserService.kt
package net.canway.service

import net.canway.pojo.tables.pojos.User
import java.util.*

interface UserService {
    fun findAllUser(): List<*>

    fun deleteByPrimaryKey(tIds: ArrayList<Int>): Int


    fun insert(record: User): Int

    fun insertSelective(record: User): Int


    fun selectByPrimaryKey(tId: Int?): User?


    fun updateByPrimaryKeySelective(record: User): Int


    fun updateByPrimaryKey(record: User): Int
}
